src/services/ProcessorService.js

```diff
@@ -44,16 +42,44 @@ function * processMessage (message) {
     logger.info(`The file ${fileName} is not in DMZ area, copying it to DMZ area.`)
     const downloadedFile = yield helper.downloadFile(message.payload.url)
     yield s3p.uploadAsync({ Bucket: config.get('aws.DMZ_BUCKET'), Key: fileName, Body: downloadedFile })
-    dmzS3Obj = yield s3p.getObjectAsync({ Bucket: config.get('aws.DMZ_BUCKET'), Key: fileName })
   }
 
-  // scan the file in DMZ
-  logger.info(`Scanning the file ${fileName}.`)
-  const form = new FormData()
-  form.append('file', dmzS3Obj.Body, { filename: fileName })
-  const scanResult = yield axios.post(config.ANTIVIRUS_API_URL, form, { maxContentLength: config.MAXFILESIZE, headers: form.getHeaders() })
+  // Send request to Scan the file
+  logger.info(`Sending request to scan the file ${fileName}.`)
+  const reqBody = {
+    submissionId: message.payload.id,
+    url: message.payload.url,
```

Comment:
@sharathkumaranbu The `url` cannot be passed from the message object here. It has to be the S3 url in DMZ storage only and thus, we cannot rely on the url from message object.